packages/stencil/components/tsconfig.components.prod.json

```diff
@@ -14,6 +14,7 @@
   "include": [
     "accordion/src",
     "cart/src",
+    "checkbox/src",
     "checkbox-button/src",
```

Comment:
- `"checkbox/src",` shoud not be here after rebasing
- remove `checkbox-button/src`